src/me/mrCookieSlime/CSCoreLibPlugin/protection/modules/PlotSquaredProtectionModule.java

```diff
@@ -4,25 +4,27 @@
 import org.bukkit.entity.Player;
 
 import com.intellectualcrafters.plot.config.C;
+import com.intellectualcrafters.plot.object.Location;
+import com.intellectualcrafters.plot.object.Plot;
 import com.intellectualcrafters.plot.object.PlotPlayer;
 import com.intellectualcrafters.plot.util.Permissions;
+
 import me.mrCookieSlime.CSCoreLibPlugin.protection.ProtectionModule;
 
 
 public class PlotSquaredProtectionModule implements ProtectionModule {
-	
+	private boolean hasAccess(Player player, Block b) {
+		Plot plot = new Location(b.getWorld().getName(), b.getX(), b.getY(), b.getZ()).getOwnedPlot();

```

Comment:
Why are you creating a new Location Object instead of using b.getLocation() ?

Reply:
It's actually a `com.intellectualcrafters.plot.object.Location`

Reply:
Ah.